applications/luci-app-adblock/root/etc/uci-defaults/40_luci-adblock

```diff
@@ -1,11 +1,4 @@
 #!/bin/sh
 
-uci -q batch <<-EOF >/dev/null
```

Comment:
Should I delete that ucitrack config like
https://github.com/openwrt/luci/blob/74cb64fbb7bf0c6a3c26ac7b0388ff0b01649d5a/applications/luci-app-ddns/root/etc/uci-defaults/40_luci-ddns#L3-L7